src/components/overview/ImageGallery/ImageGallery.jsx

```diff
@@ -2,12 +2,106 @@ import React, { useState, useEffect } from 'react';
 import DefaultView from './DefaultView.jsx';
 
 const ImageGallery = ({state, updateState}) => {
+  const [ currentStylePhotos, updateStylePhotos ] = useState([]);
+  const [ clickedImage, updateClickedImage ] = useState(0);
+
+  //update main image//
+  const updateMainImage = (event) => {
+    event.preventDefault();
+    let clickedImg = event.target.src;
+    let currentImgIndex = currentStylePhotos.indexOf(clickedImg);
+    updateClickedImage(currentImgIndex);
+
+    updateState((preValues) => {
+      return {...preValues, mainImage: clickedImg};
+    });
+  };
+
+  useEffect(() => {
+    helperForArrow();
+
+    //differenciate selected main image//
+    let currentImageElement = document.getElementById(clickedImage);
+    if (currentImageElement !== null) {
+      for (let i = 0; i < state.styleImages.length; i++) {
+        document.getElementById(i).style.borderStyle = 'none';
+        document.getElementById(i).style.opacity = '1';
+      }
+      currentImageElement.style.borderBottom = 'solid';
+      currentImageElement.style.opacity = '0.4'
+      currentImageElement.style.borderWidth = '6px';
+    }
+
+  }, [state])
+
+  //arrow click function//
+  const helperForArrow = () => {
```

Comment:
Could there be a better name for `helperForArrow`? It's a little ambiguous what exactly the function does based on it's name

Reply:
It is changed now! 👍